lib/api/2.0/nodes.js

```diff
@@ -8,9 +8,10 @@ var nodes = injector.get('Http.Services.Api.Nodes');
 var workflowApiService = injector.get('Http.Services.Api.Workflows');
```

Comment:
@srinia6 Did you mean to include this file?  Does @dalebremner's PR supersede these changes.